### DISCUSSION TITLE
Thank you !

### DISCUSSION THREAD

**Discussion by Kr1ss-XD** [dandavison/delta#466] [General]
Hey Dan,

since I coulndn't find any link or hints how to donate to you and/or the project, I figured I'd quickly step by for a "thank you".

I can doubtlessly say that delta is one of the most interesting free software projects I've encountered in this year, and which made my distro - as well as many others - more exciting. Furthermore, @dandavison you are definitely one of the most kind and welcoming FOSS developers I've ever dealt with, therefore :

https://user-images.githubusercontent.com/28707521/103096982-a7408d00-45fd-11eb-883e-c435bc103ca4.mp4

and also thanks each and everyone who has contributed to this application and to FOSS as a whole, and who took part in the communication concerning it, or helped in testing and bug wrangling.

May you and the yours have a great holiday !

**Comment by dandavison** [comment]
Thanks @Kr1ss-XD, @ulwlu, @MarcoIeni, and everyone else who has been working on Delta this year! There are still lots of ways we can improve Delta, so I'm looking forward to carrying on working with you.

**Comment by ghost** [comment]
I definetely agree with Kr1ss! Thank you dandavison for creating this great tool and your kindness.
I look forward to delta's improvement next year!
